Classes/Export/Event/AlternateExportQueryEvent.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Calien\Xlsexport\Export\Event;
+
+use Psr\EventDispatcher\StoppableEventInterface;
+
+final class AlternateExportQueryEvent implements StoppableEventInterface
+{
+    /**
+     * @var string
+     */
+    protected string $exportKey = '';
+
+    /**
+     * @var array
+     */
```

Comment:
Please describe the array a bit more, as we try to add phpstan at a later point.
```suggestion
    /**
     * @var array{
     *     export: string,
     *     manipulated?: bool
     *  }
     */
```

Reply:
More detail in 15e899cad26a2df606b5af0bff2cf8764146a5ab